Makefile

```diff
@@ -181,7 +181,7 @@ build/timestamps/check-%-timestamp: $(BUILD_HOSTED)/examples/%.html \
                                     $(BUILD_HOSTED)/build/ol.js \
                                     $(BUILD_HOSTED)/css/ol.css
 	@mkdir -p $(@D)
-	./node_modules/.bin/phantomjs --ssl-protocol=any --ignore-ssl-errors=true bin/check-example.js $<
+	./node_modules/.bin/phantomjs --web-security=no --ssl-protocol=any --ignore-ssl-errors=true bin/check-example.js $<
```

Comment:
`--local-to-remote-url-access=true` could be used instead of `--web-security=no`